templates/types/streaming/nextjs/webpack.config.o11y.mjs

```diff
@@ -1,7 +1,4 @@
 export default function webpack(config, isServer) {
-  config.resolve.fallback = {
```

Comment:
@himself65 as far as i remember, this is necessary when using MongoDB vector store, why did you remove this?